apps/dashr-financial-report/app.R

```diff
@@ -782,7 +782,7 @@ dividends_data[2:6,8] <- "-"
 
 
 
-dividends_table <- generate_table(remove.factors(dividends_data))
+dividends_table <- generate_table(dividends_data)

```

Comment:
Not sure why this diff is here -- this PR looks like it's for the Dash Bio apps. Might be a merge issue with `master`?

Reply:
I was just going through all apps I'd worked on to check the logo used on them and noticed this small bug. There's no real changes with the app itself, but should I open a separate PR for this?

Reply:
@HammadTheOne Is it possible to ignore this file and remove the directory from your commit (without erasing the existing app, of course)? Even if it triggers a conflict, we can resolve it before merging.

Reply:
Should be fixed now, I reverted the change from this commit. Is that OK?